api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>same as the above. adding a diagonal line can make it more illustrative.</p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
dibe